Coursework/2-Functions/exercise4.js

```diff
@@ -13,7 +13,10 @@ write the function
 
 // write your function here
 
+function calculateMonthsInYear(ageInMonths){
+    return ageInYears = ageInMonths * 12;
 
+}
 // Don't change any of the code below
```

Comment:
Excellent. Again, if you wanted you could just do `return ageInMonths * 12;` since it makes no difference to the computer, however one advantage of what you've done is makes it a bit easier for a human reader to understand.